tests/testthat/helper-orderly.R

```diff
@@ -0,0 +1,26 @@
+options(outpack.schema_validate =
+          requireNamespace("jsonvalidate", quietly = TRUE) &&
+          packageVersion("jsonvalidate") >= "1.4.0")
+
+
+test_prepare_orderly_example <- function(examples, ...) {
+  tmp <- tempfile()
+  withr::defer_parent(unlink(tmp, recursive = TRUE))
+  orderly_init(tmp)
+
+  config <- character()
+
+  if ("global" %in% examples) {
+    config <- c(config,
+                "global_resources: global")
+    fs::dir_create(file.path(tmp, "global"))
+    fs::file_copy("examples/minimal/data.csv", file.path(tmp, "global"))
```

Comment:
Can we wrap in `testthat::test_path` so then the tests will work interactively without changing your working directory?

```suggestion
    fs::file_copy(testthat::test_path("examples/minimal/data.csv"), file.path(tmp, "global"))
```

Reply:
yeah, absolutely

Reply:
Ugh, turns out that with this you _have_ to be in the root, I thought it worked both ways?